h2o-r/h2o-package/R/generic.R

```diff
@@ -40,5 +40,17 @@ h2o.generic <- function(model_id = NULL,
 
   # Error check and build model
   model <- .h2o.modelJob('generic', parms, h2oRestApiVersion=3, verbose=FALSE)
+    
+  # Exposing model's parameters represented as TwoDimTable as list
```

Comment:
This part should be placed into `gen_R.py`, otherwise, it'll be rewritten every time someone builds the project and triggers `gen_R.py`.